Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 ######## Update SGX SDK ########
-include UpdateRustSGXSDK.mk
+#include UpdateRustSGXSDK.mk
```

Comment:
This one is accidental, but what would you think about removing the default update at every build? It might prevent reproducible CI builds

Reply:
You mean this change is accidental? Are we sure that this has no undesired consequences, e.g. for people setting up the whole development environment for the first time?

Reply:
Anyhow. I removed it now. I just wanted to get some opinions about a default update that is done at build time.